contrib/images/Dockerfile.freebsd

```diff
@@ -0,0 +1,127 @@
+# 
+#  Copyright (C) 2024 The pgmoneta community
+# 
+#  Redistribution and use in source and binary forms, with or without modification,
+#  are permitted provided that the following conditions are met:
+# 
+#  1. Redistributions of source code must retain the above copyright notice, this list
+#  of conditions and the following disclaimer.
+# 
+#  2. Redistributions in binary form must reproduce the above copyright notice, this
+#  list of conditions and the following disclaimer in the documentation and/or other
+#  materials provided with the distribution.
+# 
+#  3. Neither the name of the copyright holder nor the names of its contributors may
+#  be used to endorse or promote products derived from this software without specific
+#  prior written permission.
+# 
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+#  EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+#  OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+#  THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+#  OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+#  HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+#  TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# 
+
+FROM freebsd:13.0
```

Comment:
Is there a `14` image, or better `:latest` ?

Reply:
I tried to use `FreeBSD` but can't download the `FreeBSD` image from Docker. Additionally, for Docker, if you want to use `FreeBSD`, the host OS must be `FreeBSD`. There is no official image in Docker for `FreeBSD` at the moment.

https://stackoverflow.com/questions/33864142/can-freebsd-be-run-inside-docker